packages/core/src/locales/zh-CN.yml

```diff
@@ -8,8 +8,10 @@ commands:
     messages:
       no-source: 当前未找到可用图源。请从插件市场添加 booru 系图源插件并启用 (插件名通常以 booru- 开头)。
       no-result: 没有找到符合条件的图片
+      no-image: 获取图片失败
       output:
         image: <message><image url={url}></image></message>
+        imgbase64: <message><image url={'data:image/png;base64,'+buffer}/></message>
```

Comment:
This is not correct since not every images are PNGs, you should not hard-code bere.
On the other hand, data URI is also valid URI, you could just produce the concatenated string in the source code and use the same i18n entry.